websites/H/HackTheBox/dist/metadata.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "HackTheBox",
+  "author": {
+    "name": "Satsujin",
+    "id": "672986823185661955"
+  },
+  "category": "games",
+  "logo": "https://i.imgur.com/ZIQbIcJ.jpg,
```

you broke metadata btw
```suggestion
  "logo": "https://i.imgur.com/ZIQbIcJ.jpg",
```

---

Crap. I'm an idiot